starlette/sessions.py

```diff
@@ -0,0 +1,170 @@
+import abc
+import uuid
+from typing import Optional
+
+try:
+    import aioredis
+except ImportError:  # pragma: nocover
+    aioredis = None
+
+try:
+    import databases
+except ImportError:  # pragma: nocover
+    databases = None
+
+try:
+    import aiomcache
+except ImportError:  # pragma: nocover
+    aiomcache = None
```

Comment:
Might be worth only (trying) to import packages for backends being used, maybe by just moving them into separate files/modules?

Reply:
I was also thinking about it. Found this solution in sterlette's templates.py